cip/Jenkinsfile

```diff
@@ -104,6 +104,22 @@ node('docker') {
             sh 'docker image save cosmotech/brewerysamplesolution_simulator:latest -o Generated/Artifact/brewerysamplesolution_simulator.tar'
           }
         }
+        stage('Generate SBOM'){
+          dir('src'){
+            sh 'curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b ~/.local/bin v0.66.0 && \
+                export PATH=$HOME/.local/bin/:$PATH && \
+                trivy image --format cyclonedx --output sbom.json cosmotech/brewerysamplesolution_simulator:latest'
+          }
+        }
+        stage('Publish SBOM to DT'){
+          dir('src'){
+            try {
+                dependencyTrackPublisher artifact: 'sbom.json', projectId: '77f1c795-86b0-48b5-9057-5c0dc397529b', synchronous: true
```

Comment:
I don't think we need the `synchronous` parameter. This is used to wait for analysis results back from Dep-Track in order to display them on the Jenkins job page/summary.
It requires additional rights, but I see that you added them to the automation key so I'm ok with this if the additional rights as been validated.